+++
title = "Workshops"
description = "Schedule for the pre-conference workshops and link to any supplementary materials"
+++

## Workshops Day 1 - Sunday 1st December 2019

### Location

Sunday Workshops will be held at the [University of Adelaide North Tce Campus](https://www.google.com/maps/place/The+University+of+Adelaide/@-34.9194115,138.6039513,18.06z/data=!4m5!3m4!1s0x6ab0ced27ef95329:0xb46ee5f583fae2bf!8m2!3d-34.920603!4d138.6062277). The listings in the location column contain a link to the location of the building on Google maps. A pdf copy of the campus map with room listings is available [here](https://www.adelaide.edu.au/campuses/mapscurrent/north_terrace.pdf). The campus is a short walk from the suggested accomodation options.

#### Parking

There is plenty of parking nearby the University, with much of it having reduced Sunday rates. The most likely place to find on-street parking is to the North of the campus on [Victoria Drive](https://www.google.com/maps/place/Victoria+Dr,+Adelaide+SA+5000/@-34.9204714,138.6038083,16.81z/data=!4m5!3m4!1s0x6ab0c92b156eea6b:0x2be42866d17959d9!8m2!3d-34.9174082!4d138.6020806), and there is plenty of high-rise parking off [North Tce](https://www.google.com/maps/dir//-34.921269,138.6056662/@-34.9192677,138.6015355,17z) or [Rundle Street](https://www.google.com/maps/dir//-34.9226321,138.6062223/@-34.9228605,138.6075143,16.81z). Note that if you are planning on using the high-rise car parking, you will need to approach from the East heading West, as is it along the southern side of North Tce, and there is a tram line which prevents turning in from the far side of the road in most cases.

<br>

<table class="cll">
<col style="width:25%">
<col style="width:40%">
<col style="width:25%">
<col style="width:10%">
<thead>
<tr>
<th>Presenter</th>
<th>Title (Click to see Abstract)</th>
<th>Location</th>
<th>Materials</th>
</tr>
</thead>

<tbody>

<tr class="clickable" data-toggle="collapse" id="1" data-target=".1collapsed">

<td>Claudia Czado</td>
<td>Analyzing dependent data with vine copulas</td>
<td><a href="https://www.google.com.au/maps/dir//-34.918762,138.6054143/@-34.9197566,138.6052004,17.57z" target=_blank>University of Adelaide North Tce Campus, Ingkarni Wardli 7.15 Conference Room</a></td>
<td><a href=""></a></td>
</tr>

<tr class="out budgets 1collapsed collapse" aria-expanded="false" style="height: 0px;">
    <td colspan="4">
        This course is designed for graduate students and researchers, who are interested in using copula based models for multivariate data structures. It provides a step to step introduction to the class of vine copulas and their statistical inference. This class of flexible copula models has become very popular in the last years for many applications in diverse fields such as finance, insurance, hydrology, marketing, engineering, chemistry, aviation, climatology and health.

<br>
<br>

The popularity of vines copulas is due to the fact, that it allows in addition to the separation of margins and dependence by the copula approach, tail asymmetries and separate multivariate component modeling. This is accommodated by constructing multivariate copulas using only bivariate building blocks, which can be selected independently. These building blocks are glued together to valid multivariate copulas by appropriate condition-
ing. Thus also the term pair copula construction was coined by Aas et al. (2009). This approach allows for flexible and tractable dependence models in dimensions of several hundred, thus providing a long desired extension of the elliptical and Archimedean copula classes. It forms the basis of new approaches in risk, reliability, spatial analysis, simulation, survival analysis and data mining to name a few.

<br>
<br>

The course starts with background on multivariate and conditional distributions and copulas. Basic bivariate dependence measures are then discussed.  Bivariate parametric classes of elliptical, Archimedean are introduced and graphical tools for the identification of sensible bivariate copula models to data are developed. The decomposition and construction principle of vines is first given in three dimensions and then extended to the special cases of draw able (D-) and canonical (C-) vines. Finally, the general case of regular (R-) vines is developed. Simulation algorithms and parameter estimation methods will be constructed. Model selection methods for vine models are considered. The short course closes with a case study. Computations are facilitated using the freely available package VineCopula of Schepsmeier et al. (2017) package in R (see R Core Team (2017)). Further resources on vine models can be found under vine-copula.org
</td></tr>

<tr class="clickable" data-toggle="collapse" id="2" data-target=".2collapsed">
<td>Emi Tanaka and Annie Conway</td>
<td>Tidyverse & R Markdown Workshop</td>
<td><a href="https://www.google.com.au/maps/dir//-34.9198926,138.6044309/@-34.9201722,138.6042604,17.86z" target=_blank>University of Adelaide North Tce Campus, Hughes 322 Teaching Room</a></td>
<td><a href=""></a></td>
</tr>

<tr class="out budgets 2collapsed collapse" aria-expanded="false" style="height: 0px;">
    <td colspan="4">
Tidyverse is a collection of R packages that share a common philosophy to perform various data science tasks. In this hands-on workshop, you will learn data wrangling and data visualisation using the ggplot2, dplyr and tidyr packages as well as touch on function programming using purrr to aid coding for repetitive tasks. 
<br><br>

R Markdown is a tool that allows you to easily integrate text, code and its output into one document. You will learn various usage of R Markdown, including producing pretty & dynamic documents, using xaringan to build html presentations, and using bookdown to build your own reproducible book.
This workshop will be suitable for people with some basic knowledge and familiarity of R. 

<br><br>
{{% md %}}

##### Biography of Instructor
Dr. Emi Tanaka has an extensive experience in using R and development of software. She has contributed to various R packages and extensions, including xaringan (html slides). Her hacking ability was praised by Dr. Yihui Xie (software engineer for RStudio, author of many R Markdown related package such as knitr, rmarkdown, xaringan and bookdown) on his [popular blog](https://yihui.name/en/2018/07/emi-tanaka/). She has given a number of tutorials and workshop related to the topics of this workshop.
{{% /md %}}
</td></tr>

<tr class="clickable" data-toggle="collapse" id="3" data-target=".3collapsed">
<td>Julian Taylor and Beata Sznajder</td>
<td>Whole Genome Analysis with wgaim</td>
<td><a href="https://www.google.com.au/maps/dir//-34.9195567,138.6045386/@-34.9197931,138.6039874,17.88z" target=_blank>University of Adelaide North Tce Campus, Barr Smith South 2052 Teaching Room</a></td>
<td><a href=""></a></td>
</tr>

<tr class="out budgets 3collapsed collapse" aria-expanded="false" style="height: 0px;">
    <td colspan="4">
        The advent of high-throughput genotyping methods has allowed the low-cost generation of high dimensional genetic marker sets for dissection and analysis. In plant and animal breeding research these analyses usually involve the incorporation of the complete set of genetic markers into a complex linear mixed model (LMM). In these models the goal is usually to determine the underlying genetic architecture of phenotypic traits through various whole genome analysis techniques such as quantitative trait loci (QTL) analysis, genome wide association studies (GWAS) or genomic selection (GS).
<br><br>
There are many LMM methods for QTL, GWAS and GS analysis. This course will provide a theoretical and computational introduction to the newly updated genetic analysis R package, <b>WGAIM</b>, that builds on the highly flexible LMM computational architecture of ASReml-R V4. The course will initially introduce LMM theory and the necessary extensions required for QTL, GWAS and GS analysis approaches. The functionality of the package will then be exemplified through a walk-through of the computational genetic analysis pipeline. Workshop participants will have a chance to gain hands on experience with the package features through practical sessions. A good working knowledge of R is assumed.

</td></tr>

</tbody>
</table>


## Workshops Day 2 - Monday 2nd December 2019

### Location

Monday Workshops will be held at the [University of Adelaide Waite Campus](https://www.google.com/maps/place/The+University+of+Adelaide+-+Waite+Campus/@-34.967,138.6338113,17z/data=!3m1!4b1!4m5!3m4!1s0x6ab0ce875dbab4d7:0xf03365545bac8d0!8m2!3d-34.967!4d138.636), except for James Carpenter's workshop which will be held in the [Adelaide Health and Medical Sciences Building](https://goo.gl/maps/1VZ8RfrytZ1HGqr79) on North Tce in the Adelaide CBD. The listings in the location column of the table below contain a link to the location of the building on Google maps. A pdf copy of the Waite campus map with room listings is available [here](https://www.adelaide.edu.au/campuses/mapscurrent/waite.pdf). The campus is a 15 minute drive or 40 minute bus ride ([on the 170 bus](https://www.google.com/maps/dir/-34.9209039,138.6054353/The+University+of+Adelaide+-+Waite+Campus,+Urrbrae+SA+5064/@-34.9439498,138.5506326,12z/data=!3m1!4b1!4m13!4m12!1m0!1m5!1m1!1s0x6ab0ce875dbab4d7:0xf03365545bac8d0!2m2!1d138.636!2d-34.967!2m3!6e1!7e2!8j1575277200!3e3)) from the Adelaide CBD, North Tce campus and the suggested accomodation options.

#### Parking

Parking at Waite is free, with ample parking places available off Waite Rd. See the [campus map](https://www.adelaide.edu.au/campuses/mapscurrent/waite.pdf) for parking locations.



<br>

<table class="cll">
<col style="width:25%">
<col style="width:40%">
<col style="width:25%">
<col style="width:10%">
<thead>
<tr>
<th>Presenter</th>
<th>Title (Click to see Abstract)</th>
<th>Location</th>
<th>Materials</th>
</tr>
</thead>

<tbody>

<tr class="clickable ws-table" data-toggle="collapse" id="4" data-target=".4collapsed">

<td><b>James Carpenter</b></td>
<td>Handling missing data in administrative studies: multiple imputation and inverse probability weighting</td>
<td><a href="https://goo.gl/maps/1VZ8RfrytZ1HGqr79" target=_blank><span style="color:#29cc85;font-weight:bold">AHMS Building, North Tce Medical Precinct<br>Level 1, Room G1059A Lecture Theatre</span></a></td>
<td><a href=""></a></td>
</tr>

<tr class="out budgets 4collapsed collapse" aria-expanded="false" style="height: 0px;">
    <td colspan="4">
        The course will consider the issues raised by missing data (both item and unit non-response) in studies using survey and routinely collected data, for example electronic health records. Following a review of the issues raised by missing data, we will focus on two methods of analysis: multiple imputation and inverse probability weighting. We will also discuss how they can be used together. The concepts will be illustrated with medical and social examples.   
</td></tr>

<tr class="clickable" data-toggle="collapse" id="5" data-target=".5collapsed">

<td>Chris Brien and Sam Rogers</td>
<td>Identifying, randomizing, canonically analyzing and formulating mixed models for designs for comparative experiments using R</td>
<td><a href="https://www.google.com.au/maps/dir//-34.9688048,138.6388042/@-34.9680507,138.6387428,16.19z" target=_blank>University of Adelaide Waite Campus<br>Plant Research Centre Auditorium</a></td>
<td><a href=""></a></td>
</tr>

<tr class="out budgets 5collapsed collapse" aria-expanded="false" style="height: 0px;">
    <td colspan="4">
        Brien (2017) outlines a mixed-model-based paradigm for obtaining A-optimal designs for comparative experiments and deriving, from the allocation involved in the design, an initial mixed model for data from an experiment that employs the design. This course explores the use of R packages <tt>od</tt> and <tt>dae</tt> for implementing this paradigm: the <tt>od</tt> package can be used, if required, to generate designs that are A-optimal for a specified mixed model; the <tt>dae</tt> package can be used to randomize designs and to perform canonical analyses (eigenanalysis) of designs. The canonical analysis is useful in elucidating the properties of a design and in formulating and checking a mixed model for it.

<br>
<br>

The course covers those basic concepts in experimental design that are necessary for using the paradigm. Methods for describing the factor allocation in a design and their use in producing a canonical analysis of the design are discussed, along with interpreting the canonical analysis. The formulation of allocation-based mixed models from the canonical analysis is also exposited. That is, the trail from the recognition in the planning stages of important sources of variation through constructing the design to the mixed model for data from the experiment using the design is followed. Participants will rehearse the techniques in practical sessions, using the R packages <tt>dae</tt> and <tt>od</tt>.
</td></tr>





<tr class="clickable" data-toggle="collapse" id="6" data-target=".6collapsed">

<td>Peter Kasprzak and Kathy Ruggiero</td>
<td>Shiny App Development</td>
<td><a href="https://www.google.com.au/maps/dir//-34.9680263,138.6351788/@-34.9671704,138.6360118,16.34z" target=_blank>University of Adelaide Waite Campus<br>Biometry Hub Meeting room<br>Room 204 Waite Building</a></td>
<td><a href=""></a></td>
</tr>

<tr class="out budgets 6collapsed collapse" aria-expanded="false" style="height: 0px;">
    <td colspan="4">
        Shiny is a tool purpose-built for the R programming language that makes it quick and easy to create interactive web applications for data exploration. By leveraging the full power of R, apps can quickly be prototyped from your existing scripts with very little additional overhead.  
<br><br>
In this short course, we will approach Shiny from a beginner's perspective, walking through the basics of how a Shiny application works and how to build one. Towards the conclusion of the workshop, we will discuss methods for publishing and distributing applications, as well as more advanced techniques for building Shiny programs in a future-proof and maintainable fashion. A basic level of R knowledge is assumed.

</td></tr>


<tr class="clickable" data-toggle="collapse" id="7" data-target=".7collapsed">
<td>Christopher K. Wikle and Dan Pagendam</td>
<td>An Introduction to Deep Learning with Biometric and Environmetric Applications</td>
<td><a href="https://www.google.com.au/maps/dir//-34.9688661,138.6376184/@-34.9662941,138.6380513,16.13z" target=_blank>University of Adelaide Waite Campus<br>Plant Genomics Centre Room 1.27</a></td>
<td><a href=""></a></td>
</tr>

<tr class="out budgets 7collapsed collapse" aria-expanded="false" style="height: 0px;">
    <td colspan="4">
        Deep learning is a type of machine learning (ML) that exploits a connected hierarchical set of models to predict or classify elements of complex data sets. The ML deep learning revolution is relatively recent and primarily associated with neural models such as feedforward neural networks (FNNs), convolutional neural networks (CNNs), recurrent neural networks (RNNs), generative adversarial networks (GANs), or some combination of these neural architectures. There are remarkable success stories associated with these approaches, such as models that can defeat experts in Go, Chess, or Shogi, and of course, there are failures as well. Statisticians should not be surprised by the success (and failure) of these deep ML methods as we have been using deep hierarchical models (HMs) for years. Indeed, many of the reasons for success and failure of deep ML and deep HMs are the same. 
     
     <br><br>
     
This course will present an introduction to deep models in ML from a statistician’s perspective. Topics will include an introduction to stochastic gradient optimization and concepts in regularization and dimension reduction, followed by discussion of deep FNNs, CNNs, and RNNs. We will also touch upon some recent developments that may be of particular interest to statisticians.  The course will focus on concepts and modeling intuition, and will include hands-on implementation using the R interface to Keras, with examples from biomedical, ecological, and environmental statistics. 


</td></tr>

</tbody>
</table>
